src/pages/LegacySeries/utils.ts

```diff
@@ -6,11 +8,17 @@ import type { EpisodeMetadata } from '#types/series';
 /**
  * Get an array of options for a season filter
  */
-export const getFiltersFromSeries = (playlist: Playlist | undefined): string[] => {
+export const getFiltersFromSeries = (playlist: Playlist | undefined, t: TFunction): string[] => {
```

Comment:
You don't need to pass `t` in. You can either get it from an import like this: https://github.com/jwplayer/ott-web-app/blob/develop/src/services/config.service.ts#L114
or you can convert this to a hook if there's any concern that the filters won't regenerate when the language get's switched.